exec/plugin-commands-script-runners/src/exec.ts

```diff
@@ -120,6 +127,19 @@ export async function handler (
       }
     }
   }
+
+  if (opts.resumeFrom) {
+    const resumeFromPackagePrefix = Object.keys(opts.selectedProjectsGraph)
+      .find((prefix) => opts.selectedProjectsGraph?.[prefix]?.package.manifest.name === opts.resumeFrom)
+
+    if (!resumeFromPackagePrefix) {
+      throw new PnpmError('RECURSIVE_EXEC_FAIL', `Cannot find package ${opts.resumeFrom}. Could not determine where to resume from.`)
+    }
+
+    const chunkPosition = chunks.findIndex(chunk => chunk.includes(resumeFromPackagePrefix))
```

Comment:
Shouldn't this be an equal check?
```suggestion
    const chunkPosition = chunks.findIndex(chunk => chunk === resumeFromPackagePrefix)
```

Reply:
`chunks` looks like a two-dimensional array, for example:

```
[
  [
    'packages/bar',
    '/packages/foo'
  ],
  [
    'packages/qar',
    '/packages/foo'
  ],
]
```

`chunkPositio`n I use to determine the index of this package in the `chunks` array.
Since the packages inside the chunk will be executed in parallel, it is not easy to determine the order, so i think it seems to be possible to re-execute directly from the chunkPosition.